app/models/team.rb

```diff
@@ -53,6 +53,10 @@ def by_season_phase
         Team.visible.where season: Season.current
       end
     end
+
+    def active_teams
+      Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
```

Comment:
I was hoping you would say something like that :-) .  Maybe I better open a new PR for that, wdyt?